osu.Game/Storyboards/Drawables/DrawableStoryboardSprite.cs

```diff
@@ -10,13 +10,20 @@
 using osu.Framework.Graphics.Textures;
 using osu.Framework.Utils;
 using osu.Game.Beatmaps;
+using osu.Game.Skinning;
 
 namespace osu.Game.Storyboards.Drawables
 {
-    public class DrawableStoryboardSprite : Sprite, IFlippable, IVectorScalable
+    public class DrawableStoryboardSprite : SkinReloadableDrawable, IFlippable, IVectorScalable
     {
         public StoryboardSprite Sprite { get; }
 
+        private Sprite drawableSprite;
+
+        private TextureStore storyboardTextureStore;
```

Comment:
pretty sure you could make this a

```suggestion
        [Resolved]
        private TextureStore storyboardTextureStore { get; set; }
```

instead of adding this local and assigning to it in BDL

Reply:
Yep, that works